src/platforms/eglstream-kms/server/platform_symbols.cpp

```diff
@@ -163,7 +163,7 @@ auto probe_display_platform(
     }
 
     std::vector<mg::SupportedDevice> supported_devices;
-    for (auto i = 0; i <= device_count; ++i)
+    for (auto i = 0; i != device_count; ++i)
```

Comment:
I'd generally spell this as `<` rather than `!=`, but that's just out of boundless paranoia.

Sadly, that paranoia doesn't protect one from off-by-one errors. :woman_facepalming: